txn.go

```diff
@@ -388,6 +388,41 @@ func (txn *Txn) Range(fn func(idx uint32)) error {
 	return nil
 }
 
+// SortedRange ascends through a given SortedIndex and returns each offset
+// remaining in the transaction's index
+func (txn *Txn) SortedRange(sortIndexName string, fn func(idx uint32)) error {
```

Comment:
Would it be possible to have 2 functions `Ascend()` and `Descend()` instead? It's unclear otherwise which direction the range will be done.